config/management/operational/src/test_helper.rs

```diff
@@ -295,6 +305,7 @@ impl OperationalTool {
         self.rotate_key(
             backend,
             disable_validate,
+            false,
```

Comment:
I think we might want to either not validate the addresses, or allow it to be configurable.  Makes sense to me for this to be `true` because the operator key doesn't affect the network addresses (therefore if the addresses were wrong already, we shouldn't stop operator key rotation)

Reply:
See my comment below.